src/lib/objects.js

```diff
@@ -0,0 +1,52 @@
+const createPerson = (name, age) => {
```

Comment:
As above, should adad a note to the readme about your intention to complete these later